services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java

```diff
@@ -290,54 +300,54 @@ private void verifyRowExistsInLocalTable(int expectedNumRows, BaseTable baseTabl
-             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs("test2", 15, 3.1415));
+             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs(TEST_STRING_ONE, TEST_INT_TWO, TEST_DECIMAL_ONE));
          serviceInterface
-             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs("test", 15, 3.1415));
+             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs(TEST_USERNAME, TEST_INT_TWO, TEST_DECIMAL_ONE));
 
          serviceInterface
-             .updateLocalOnlyRows(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs("test", 16, 3.1415),
-                 "columnId = ?", new BindArgs(new String[] { "test" }));
+             .updateLocalOnlyRows(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs(TEST_USERNAME, TEST_INT_ONE, TEST_DECIMAL_ONE),
+                     TEST_STRING_TWO, new BindArgs(new String[] { TEST_USERNAME }));
 
          BaseTable baseTable = serviceInterface
              .simpleQueryLocalOnlyTables(APPNAME, dbHandle, TABLE_LOCAL, null, null, null, null,
                  null, null, null, null);
 
-         verifyRowExistsInLocalTable(2, baseTable, "test", 16, 3.1415);
-         verifyRowExistsInLocalTable(2, baseTable, "test2", 15, 3.1415);
+         verifyRowExistsInLocalTable(2, baseTable, TEST_USERNAME, TEST_INT_ONE, TEST_DECIMAL_ONE);
+         verifyRowExistsInLocalTable(2, baseTable, TEST_STRING_ONE, TEST_INT_TWO, TEST_DECIMAL_ONE);
 
          ContentValues cv = new ContentValues();
-         cv.put(COLUMN_ID3, 9.5);
+         cv.put(COLUMN_ID3, TEST_DECIMAL_TWO);
          serviceInterface.updateLocalOnlyRows(APPNAME, dbHandle, TABLE_LOCAL, cv, "columnId2 = ?",
              new BindArgs(new Object[] { 16 }));
 
          baseTable = serviceInterface
              .simpleQueryLocalOnlyTables(APPNAME, dbHandle, TABLE_LOCAL, null, null, null, null,
                  null, null, null, null);
 
-         verifyRowExistsInLocalTable(2, baseTable, "test", 16, 9.5);
-         verifyRowExistsInLocalTable(2, baseTable, "test2", 15, 3.1415);
+         verifyRowExistsInLocalTable(2, baseTable, TEST_USERNAME, TEST_INT_ONE, TEST_DECIMAL_TWO);
+         verifyRowExistsInLocalTable(2, baseTable, TEST_STRING_ONE, TEST_INT_TWO, TEST_DECIMAL_ONE);
 
          serviceInterface
-             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs("a", 2, 3.1415));
+             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs(TEST_STRING_THREE, 2, TEST_DECIMAL_ONE));
          baseTable = serviceInterface
              .simpleQueryLocalOnlyTables(APPNAME, dbHandle, TABLE_LOCAL, null, null, null, null,
                  null, null, null, null);
 
-         verifyRowExistsInLocalTable(3, baseTable, "test", 16, 9.5);
-         verifyRowExistsInLocalTable(3, baseTable, "test2", 15, 3.1415);
-         verifyRowExistsInLocalTable(3, baseTable, "a", 2, 3.1415);
+         verifyRowExistsInLocalTable(3, baseTable, TEST_USERNAME, TEST_INT_ONE, TEST_DECIMAL_TWO);
```

Comment:
The TEST_USERNAME is not the same conceptually as TEST_STRING. While the values might be the same it should be kept separate as if we change the username, we may not want to change the test string.

Reply:
Okay. I have made the change. Thanks